tools/concepts/mfcc/test_mfcc.m

```diff
@@ -0,0 +1,219 @@
+% Script to draft MFCC (Mel frequency cepstral coefficients) audio features
```

Comment:
I'm not sure what would be a suitable location for Matlab concepts storing. I used here tools but it could be other too.

Reply:
Looks good here as prep work for a concept